### PR TITLE
feat(support-las-v3): parsing las files version 3 has been added

### PR DESCRIPTION
**Pull request by DmitroSamoilenko**
Parsing LAS files version 3 has been added

### CODE REVIEW

**Comment by iykekings** [comment]
@DmitroSamoilenko thanks for the PR

File: src/__test__/sample_v3/E3.las [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+
+~Parameter
+#MNEM.UNIT              Value                              Description
+#---------  ----------------------------------------   -----------------------------------
+RUN .                                         1  : Run number
+GL  .Ft                             1363.000000  : Ground level elevation
+DREF.                                        KB  : Depth reference
+
+~Other
```
Comment (iykekings) [inline src/__test__/sample_v3/E3.las:51]: I thought there is no ~Other section is v3 specification?

Comment (DmitroSamoilenko) [reply src/__test__/sample_v3/E3.las:51]: I did not find an example with `~Other`, but I don't see a problem because ~Other is optional.

Comment (DmitroSamoilenko) [reply src/__test__/sample_v3/E3.las:51]: Should I remove this ~Other sections from examples?

Comment (iykekings) [reply src/__test__/sample_v3/E3.las:51]: Yeah that's alright cos ~Other is removed in v3

**Comment by iykekings** [comment]
@DmitroSamoilenko can you push some commits, could be an empty one, so the CI can run?

**Comment by DmitroSamoilenko** [comment]
Hi @iykekings, ok I have removed the other section, and pushed it.

**Comment by DmitroSamoilenko** [comment]
Hi @iykekings, commits did not run CI.

**Comment by iykekings** [comment]
Okay bump the version to 3.4.0 and I will merge. I have verified that all the tests passes

**Comment by DmitroSamoilenko** [comment]
@iykekings Hello, yes I have update the version.

**Pull request merged**